src/components/screens/PointOfInterest.js

```diff
@@ -151,6 +154,15 @@ export const PointOfInterest = ({ data, hideMap, navigation }) => {
           </View>
         )}
 
+        {!!lunches?.length && (
+          <WrapperVertical>
+            <Button
+              title={texts.pointOfInterest.showLunches}
+              onPress={() => navigation.push('Lunch', { title: texts.widgets.lunch, poiId: id })}
+            />
+          </WrapperVertical>
+        )}
```

Comment:
the button seems to wide on portrait

![Bildschirmfoto 2021-02-09 um 18 04 00](https://user-images.githubusercontent.com/1942953/107399603-33046f00-6b01-11eb-847e-9e90fd70bd8b.png)

Reply:
replaced wrappervertical with wrapper in 3795523c9fe7f462df6678bc1c684c0c58969914

Reply:
so now it looks correct? there is a spaceing left and right to the phone edge?